server/src/Packages/Paypal/Resources/views/payment.blade.php

```diff
@@ -0,0 +1,14 @@
+@extends('Paypal\Payment::app')
+
+@section('content')
+<div class="uk-container uk-container-xsmall uk-margin-large-top">
+    <h1 class="uk-heading-small uk-padding-small">Complete purchase</h1>
+</div>
+<div id="stripeFormTemplate" class="uk-container uk-container-xsmall uk-margin-large-top">
```

Comment:
@AlexeiaW also reference here. Looks like you do need this div, so you can just change the naming to `paypalFormTemplate`

Reply:
renamed thank you